### ISSUE TITLE
Alignment Issue in Cloud Storage page

### ISSUE THREAD

**Issue by thekavikumar** [opencv/cvat#7657]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://opencv.github.io/cvat/docs/)

### Steps to Reproduce

![image](https://github.com/opencv/cvat/assets/50093149/f8c0b28e-bdf0-4725-bf1a-089a4f7323b9)

The component should be centered from both x axis and y axis, but it is centered only in x axis.

### Expected Behavior

The empty component should be center in the center from both the axis

**Comment by syedbarimanjan** [comment]
> ### Actions before raising this issue
> * [x]  I searched the existing issues and did not find anything similar.
> * [x]  I read/searched [the docs](https://opencv.github.io/cvat/docs/)
> 
> ### Steps to Reproduce
> ![image](https://private-user-images.githubusercontent.com/50093149/315918105-f8c0b28e-bdf0-4725-bf1a-089a4f7323b9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEwOTQwNTAsIm5iZiI6MTcxMTA5Mzc1MCwicGF0aCI6Ii81MDA5MzE0OS8zMTU5MTgxMDUtZjhjMGIyOGUtYmRmMC00NzI1LWJmMWEtMDg5YTRmNzMyM2I5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIyVDA3NDkxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM4MjZjZWRkN2FhOTZiMjY2YTg1YzVlMmU0YjRiMmY5MWQ2ZTU3NDczNmViNjAwNzkyNzc1MDlkODQ4MjdkMGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.aoibzSTYU48XBBHpXGRWp9cSlJUNo4eHJLh6wYBD058)
> 
> The component should be centered from both x axis and y axis, but it is centered only in x axis.
> 
> ### Expected Behavior
> The empty component should be center in the center from both the axis
> 
> ### Possible Solution
> _No response_
> 
> ### Context
> _No response_
> 
> ### Environment
> _No response_

I have a pr for this and it is big I am refactoring the code and will push it just to make sure that both of us dont make the same changes.

**Comment by thekavikumar** [comment]
@syedbarimanjan did you not create issue?

**Comment by syedbarimanjan** [comment]
> @syedbarimanjan did you not create issue?

I was about to create it. sorry.

**Comment by syedbarimanjan** [comment]
> @syedbarimanjan can you please allow me to work on, I have very less pr though

Well I am already done with it i have made other changes also to the emptylist components.

**Comment by thekavikumar** [comment]
@syedbarimanjan nvm, if you have already done with it, make a pr on this issue itself

**Comment by thekavikumar** [comment]
@syedbarimanjan but make sure to create issue first and then work on it

**Comment by syedbarimanjan** [comment]
> @syedbarimanjan nvm, if you have already done with it, make a pr on this issue itself

Yeah i am creating a draft pr as i just found some other problems.

**Comment by syedbarimanjan** [comment]
> @syedbarimanjan but make sure to create issue first and then work on it

I found some bug In the projects page and will make an issue you can work on that.

**Comment by thekavikumar** [comment]
@syedbarimanjan sure, thanks

**Comment by thekavikumar** [comment]
@syedbarimanjan please tag me on those issues
